scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -121,6 +179,11 @@ def discover(self):
             raise TwisterRuntimeError("No test cases found at the specified location...")
 
         self.find_subtests()
+        # get list of scenarios we have parsed into one list
+        for _, ts in self.testsuites.items():
+            self.scenarios.append(ts.id)
```

Comment:
do we need check the uniq id here? I guess same cases maybe include multiply times

Reply:
yes, I guess we can do the uniqueness check here.